dialogy/plugins/text/classification/xlmr.py

```diff
@@ -112,6 +111,8 @@ def __init__(
                     f"'best_model_dir' missing in passed args_map."
                 )
             
+            self.ts_parameter:float = read_from_json([const.TS_PARAMETER], self.model_dir, const.CALIBRATION_CONFIG_FILE).get(const.TS_PARAMETER) or self.args_map.get("ts_parameter") or 1.0
```

Comment:
Won't work when purpose is `PRODUCTION` . Put it outside the if block (just above it)